config/initializers/assets_paths.rb

```diff
@@ -0,0 +1,20 @@
+=begin
```

Comment:
@alongoldboim Use plain old # comments, rubocop is failing [because of this block](https://github.com/bbatsov/ruby-style-guide#no-block-comments)